text/0340-firehose-l2.md

```diff
@@ -0,0 +1,944 @@
+---
+rfc pr: [#342](https://github.com/aws/aws-cdk-rfcs/pull/342)
+tracking issue: [#340](https://github.com/aws/aws-cdk-rfcs/issues/340)
+---
+
+# Amazon Kinesis Data Firehose Delivery Stream L2
+
+The `aws-kinesisfirehose` construct library allows you to create Amazon Kinesis
+Data Firehose delivery streams and destinations with just a few lines of
+code. As with most construct libraries, you can also easily define permissions
+and metrics using a simple API.
+
+The amount of effort needed to create these resources is about the same as doing
+it using the AWS console and you can choose from any of the supported
+destinations.
+
+## Working Backwards
+
+### CHANGELOG
+
+`feat(kinesisfirehose): DeliveryStream L2; S3, Elasticache, Redshift destinations`
+
+### README
+
+---
+
+# Amazon Kinesis Data Firehose Construct Library
+
+[Amazon Kinesis Data Firehose](https://docs.aws.amazon.com/firehose/latest/dev/what-is-this-service.html)
+is a service for fully-managed delivery of real-time streaming data to storage services
+such as Amazon S3, Amazon Redshift, Amazon Elasticsearch, Splunk, or any custom HTTP
+endpoint or third-party services such as Datadog, Dynatrace, LogicMonitor, MongoDB, New
+Relic, and Sumo Logic. This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk)
+project. It allows you to create Kinesis Data Firehose delivery streams.
+
+## Creating a Delivery Stream
+
+In order to create a Delivery Stream, you must specify a destination. An S3 bucket can be
+used as a destination. More supported destinations are covered [below](#destinations).
+
+```ts
+import * as destinations from '@aws-cdk/aws-kinesisfirehose-destinations';
+
+new DeliveryStream(this, 'Delivery Stream', {
+  destination: new destinations.S3(),
```

Comment:
The main use case is that after dumping all of your data to some storage, S3 Bucket or otherwise, then you would perform some kind of big data analysis on it. So in some cases, humans might look in the bucket contents, but in most cases I think the volume of data will be too large for that. 

Considering that, I think you are right that it would make more sense to make the Bucket a required parameter.